x-pack/plugins/security_solution/public/management/components/endpoint_responder/components/offline_callout.tsx

```diff
@@ -22,31 +22,23 @@ interface OfflineCalloutProps {
 }
 
 export const OfflineCallout = memo<OfflineCalloutProps>(({ agentType, endpointId, hostName }) => {
-  const isEndpointAgent = agentType === 'endpoint';
-  const isSentinelOneAgent = agentType === 'sentinel_one';
   const isSentinelOneV1Enabled = useIsExperimentalFeatureEnabled(
     'responseActionsSentinelOneV1Enabled'
   );
 
-  const { data: endpointDetails } = useGetEndpointDetails(endpointId, {
-    refetchInterval: 10000,
-    enabled: isEndpointAgent,
+  const { data } = useAgentStatus([endpointId], agentType, {
+    refetchInterval: DEFAULT_POLL_INTERVAL,
+    enabled: agentType === 'endpoint' || (agentType === 'sentinel_one' && isSentinelOneV1Enabled),
```

Comment:
can I suggest that we instead do `RESPONSE_ACTION_AGENT_TYPE.includes(agentType)` here?

We don't seem to be checking for errors, so if the FF is not enabled and the API call fails, then it should not impact the UI (you can test it out).

I think that once we refactor the `isResponseActionSupported()` methods we can probably also account for the FF being enabled/disabled and then use that common service function here instead.

Reply:
Makes sense. The responder won't be shown to sentinelOne alert when FF is disabled so in that case the API won't be called anyway.

12b0d8a8461d1fad2e363f81dc786aa7853ade87